lib/sheet.js

```diff
@@ -0,0 +1,76 @@
+import Spreadsheet from 'edit-google-spreadsheet';
+import googleAuth from './google-auth';
+import util from './util';
+
+let SHEET_CONFIG;
+util.readJsonFromFile('./google-spreadsheet-config.json', data => {
```

Comment:
`util.readJsonFromFile` is an async method and there's no guarantee it will finish running before you use its results (`SHEET_CONFIG`) later in the code... You gotta use some async techniques around it to get it safe: either plain old callback, or async libraries, or promises, or generators...